pydocstringformatter/_configuration/arguments_manager.py

```diff
@@ -148,6 +154,14 @@ def parse_options(
         if self.namespace.style is None:
             self.namespace.style = ["pep257"]
 
+        for formatter in self.formatters:
```

Comment:
Why is this additional looping necessary?

Reply:
So that [the default value for any given formatter](https://github.com/DanielNoord/pydocstringformatter/pull/132#issuecomment-1200207883) depends on whether `formatter.style in self.namespace.style`, the latter of which is not known when setting default values of the parameters, but only at runtime.

That is, so that the PEP257 formatters are on by default for `--style=pep257` and `--style=pep257 --style=numpydoc` but not `--style=numpydoc`, in a way that allows for `--style=pep257 --no-quotes-type`, `--style=numpydoc --no-name-type-spacing`, or `--style=numpydoc --final-period`.

I don't see another way to have all the formatter activation values be booleans at the end of `parse_options` and also have all the default values, set in `register_arguments_formatters` in `__init__`, be dependent on the value of `--style`, determined in `parse_command_line_arguments` and three lines up.